build.py

```diff
@@ -30,10 +31,13 @@
 DEFAULT_CONFIG = "etc/kapacitor/kapacitor.conf"
 PREINST_SCRIPT = None
 
+# Default AWS S3 bucket for uploads
+DEFAULT_BUCKET = "kapacitor"
```

Comment:
I have been using the influxdb bucket. To my knowledge there is no kapacitor bucket.